txstratum/cli.py

```diff
@@ -28,6 +28,7 @@ def create_parser() -> ArgumentParser:
     parser.add_argument('--tx-timeout', help='Tx mining timeout (seconds)', type=int, default=None)
     parser.add_argument('--fix-invalid-timestamp', action='store_true', help='Fix invalid timestamp to current time')
     parser.add_argument('--prometheus', help='Path to export metrics for Prometheus', type=str, default=None)
```

Comment:
Maybe rename to `--prometheus-path`?

Reply:
I didn't do this to avoid adding a breaking change. Someone else could be using this arg out there and would break when upgrading.